### DISCUSSION TITLE
Translation into German - Formal or informal?

### DISCUSSION THREAD

**Discussion by utzaki** [nielsmaerten/gluco-check#151] [General]
@nielsmaerten In the German language, we have formal speaking ("Geben Sie den Wert ein.") and informal speaking ("Gib den Wert ein"). What would you like to use in Gluco Check for translations when addressing the user? I suggest informally as Google usually also answers in informal language when talking to the user.

**Comment by nielsmaerten** [comment]
@utzaki I agree! In Dutch, there's a similar formal/informal distinction. I've also used informal language when I translated Gluco Check into Dutch.

**Comment by utzaki** [comment]
@nielsmaerten Good! I already did the full translation to the German language yesterday by using informal language. So I do not need to change it. Nevertheless, tonight I will double-check all translations, so tomorrow you can use them.
